app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheetController.kt

```diff
@@ -88,13 +95,44 @@ class ScheduleAppointmentSheetController @Inject constructor(
   }
 
   private fun schedulingSkips(events: Observable<UiEvent>): Observable<UiChange> {
-    return events.ofType<SchedulingSkipped>()
+    val patientUuidStream = events
+        .ofType<ScheduleAppointmentSheetCreated>()
+        .map { it.patientUuid }
+
+    val combinedStreams = Observables.combineLatest(
+        events.ofType<SchedulingSkipped>(),
+        patientUuidStream,
+        configProvider.toObservable()
+    )
+    val defaulterFeatureEnabledStream = combinedStreams
```

Comment:
This name implies that this is a stream that emits whether the feature is enabled or not. Rename this stream to `closeSheetWithDefaulterFeatureEnabled` and the other `closeSheetWithDefaulterFeatureDisabled`?

Reply:
These stream names are created to distinguish between the two streams. 
The stream name `defaulterFeatureEnabledStream` here means that this stream deals with defaulter feature being enabled whereas the other one deals with the feature being disabled.